### ISSUE TITLE
rename table with sql code is not working

### ISSUE THREAD

**Issue by RosaIta** [sqlitebrowser/sqlitebrowser#2563]
Hello everyone,

I am testing the nightly windows versions (zip) both on Windows 10 and Windows 8.1:
DB Browser for SQLite Version 3.12.99 (Jan 27 2021)
Compilato per x86_64-little_endian-llp64, in esecuzione su x86_64
Qt Version 5.12.8
Versione SQLite 3.34.1

Whenever I write this SQL code:
create table products (
id integer not null primary key,
name text);

alter table products rename to vegetables;

The new name (vegetables) is not applied to the table producsts in the GUIs even though the message of the SQL window confirms this change. If I take a look at the schema in the GUIs the old name products is still present.
If I check the table: 
select * from vegetables;
an error message appears (table not existing).

Here is a screenshot on Windows 8.1:
![immagine](https://user-images.githubusercontent.com/78258259/106354416-f1e6b080-62f1-11eb-8018-0ad412b266bd.png)

I have checked the stable version (Windows 10 - 8.1) and everything is working as expected. 
Therefore I suppose it is a regression (at least on Windows 10 - 8.1):
DB Browser for SQLite Versione 3.12.1
Compilato per x86_64-little_endian-llp64, in esecuzione su x86_64
Versione Qt 5.12.8
Versione SQLite 3.33.0

In short, with the stable version, on Windows, I can rename my table through the SQL code *AND* the new name is *also* applied in the GUIs as well.

**Comment by chrisjlocke** [comment]
Aah, the nightlies and the main release are two different code bases (well, slightly diverged over time) where parts are tweaked and developed in the nightlies, and once working, cherry-picked to go in the main release.  I assume the tweak and 'fix' put into the main release was broken in the nightlies.  It's handy you raise this, as these little things sometimes point to something else 'broken'.

**Comment by RosaIta** [comment]
There is also a very similar bug:
alter table products rename column name to surname
The surname column is *not* applied in the GUIs even though the message in the Sql window confirms this change

As above, this bug is only present in the nighlies (the stable version works fine). 
Tested on Windows 10 - 8.1

To recap: 
SQL code to rename Tables and columns (nighlies builds only) does not work

**Comment by chrisjlocke** [comment]
> SQL code to rename Tables and columns (nightlies builds only) does not work

Just to be clear, the SQL code *does* work, but the GUI is not updated to reflect the changes.
What happens if you switch tabs?  From my understanding (haven't tested this yet) but when you switch tabs the schema is sometimes re-read.  I'll have a go later with the latest nightly.  I know things were tweaked as the nightlies have the multi-docked windows which the main release doesn't have.

**Comment by RosaIta** [comment]
Hello @chrisjlocke 

>  What happens if you switch tabs? 

Nothing. The "old" name for the table is still present (which is: "products").
I have also restarted the software to no avail...

To change the name of the table "products" (or the name of its columns) I am forced to use the GUIs (not the SQL code):
![immagine](https://user-images.githubusercontent.com/78258259/106358220-0a16f980-630b-11eb-9be7-8e30fd2b6687.png)

**Comment by chrisjlocke** [comment]
Aah, apologies - thought it was working, but the GUI wasn't updating.
Bit weird how its reporting no error, although it is saying, "Execution finished without errors.
Result: 0 rows returned in 7ms" - as if we're parsing it, but not actually executing it, then thinking its been executed.

Thanks for reporting it!

**Comment by MKleusberg** [comment]
This bug  should be fixed in tomorrow's nightly build. Can you give that another try, @RosaIta? :smile:

**Comment by RosaIta** [comment]
Hello @MKleusberg 

Thanks a lot: now it works!
Tested on Windows 10, with DB Browser for SQLite Version 3.12.99 (Mar 16 2021)

**Comment by chrisjlocke** [comment]
Thanks for confirming, and raising the issue in the first place.  Your assistance has made DB4S better!

**Comment by MKleusberg** [comment]
Awesome! Thanks for the quick feedback, @RosaIta!
